viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -25,6 +49,21 @@ config from_viam_carto_config(viam_carto_config vcc) {
     c.map_rate_sec = vcc.map_rate_sec;
     c.mode = vcc.mode;
     c.lidar_config = vcc.lidar_config;
+    if (c.sensors.size() == 0) {
+        throw VIAM_CARTO_SENSORS_LIST_EMPTY;
+    }
+    if (c.data_dir.size() == 0) {
+        throw VIAM_CARTO_DATA_DIR_EMPTY;
+    }
```

Comment:
Should this be called something like `VIAM_CARTO_DATA_DIR_NOT_PROVIDED`?  `VIAM_CARTO_DATA_DIR_EMPTY` makes me think that the directory provided has nothing in it which would not be an issue right?

Reply:
Great suggestion! Done.